### DISCUSSION TITLE
How to remove all git lfs files and downgade the plan

### DISCUSSION THREAD

**Discussion by ando-takahiro** [community/community#21890] [New to GitHub]
<p>Hi, I’d like to stop using git lfs.  I removed lfs from my repositories and pushed.  However, I could not reduce the storage size at <code>Billing overview</code> &gt; <code>Git LFS Data</code>.  How can I cancel the payment plan of git lfs?</p>
<p>Thanks,</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/ando-takahiro">@ando-takahiro</a>,</p>
<p>It may be that there are some objects in your repo that are keeping you over the 1GB limit. I’d recommend reaching out to<a>private support</a> so they can take a closer look for you and help you with that downgrade. You can reference this forum thread in your message so you don’t have to write out your question again.</p>
<p>Cheers!</p>

**Comment by ando-takahiro** [comment]
<p>Thanks!  I’ll look into and ask support.</p>
